lib/Fetcher/FeedFetcher.php

```diff
@@ -325,18 +325,22 @@ protected function getFavicon(FeedInterface $feed, string $url)
     {
         $favicon = $feed->getLogo();
 
+        ini_set('user_agent', 'NextCloud-News/1.0');
+
         // check if feed has a logo
         if (is_null($favicon)) {
             return $this->faviconFactory->get($url);
         }
 
         $favicon_path = join("/", [$this->ITempManager->getTempBaseDir(), basename($favicon)]);
-        copy(
+
+        $downloaded = copy(
```

Comment:
I still get errors `copy(): Filename cannot be empty at /var/www/nextcloud/apps/news/lib/Fetcher/FeedFetcher.php#340`; perhaps that should be addressed as well here.

Reply:
Maybe the new change fixes that